src/viewPanel/tabView.ts

```diff
@@ -25,12 +38,17 @@ export class TabView extends Widget {
 
     this._addCommands();
 
-    this._model.ready.connect(() => {
-      this._initGridItems();
+    this._ready = new Signal<this, null>(this);
+    this._ready.connect(() => {
+      this._updateViewers();
+    });
+
+    this._context.sessionContext.ready.then(() => {
+      this._ready.emit(null);
     });
```

Comment:
`this._ready` is private without a getter and is not used elsewhere. 

```suggestion
    this._context.sessionContext.ready.then(() => {
      this._updateViewers();
    });
```